modules/Bio/EnsEMBL/Analysis/Hive/Config/HomologyRnaseq.pm

```diff
@@ -0,0 +1,322 @@
+    'registry_db'                     => '', # name for registry db
+    'release_number'                  => '' || $self->o('ensembl_release'),
+    'species_name'                    => '', # e.g. mus_musculus
+    'production_name'                 => '', # usually the same as species name but currently needs to be a unique entry for the production db, used in all core-like db names
+    'taxon_id'                        => '', # should be in the assembly report file
+    'uniprot_set'                     => '', # e.g. mammals_basic, check UniProtCladeDownloadStatic.pm module in hive config dir for suitable set,
+    'use_genome_flatfile'             => '1',# This will read sequence where possible from a dumped flatfile instead of the core db
+    'output_path'                     => '', # Lustre output dir. This will be the primary dir to house the assembly info and various things from analyses
+    'skip_rnaseq'                     => '0', # Will skip rnaseq analyses if 1
+
+    # Keys for custom loading, only set/modify if that's what you're doing
+    load_toplevel_only                => '1', # This will not load the assembly info and will instead take any chromosomes, unplaced and unlocalised scaffolds directly in the DNA table
+    custom_toplevel_file_path         => '', # Only set this if you are loading a custom toplevel, requires load_toplevel_only to also be set to 2
+    use_repeatmodeler_to_mask         => '0', # Setting this will include the repeatmodeler library in the masking process
+    red_logic_name                    => 'repeatdetector', # logic name for the Red repeat finding analysis
+    replace_repbase_with_red_to_mask  => '0', # Setting this will replace 'full_repbase_logic_name' with 'red_logic_name' repeat features in the masking process
+
+########################
+# Pipe and ref db info
+########################
+
+    'pipe_db_name'                  => $self->o('dbowner').'_'.$self->o('production_name').'_pipe_'.$self->o('release_number'),
+    'dna_db_name'                   => $self->o('dbowner').'_'.$self->o('production_name').'_core_'.$self->o('release_number'),
+
+    'genblast_db_server'           => $self->o('databases_server'),
+    'genblast_db_port'             => $self->o('databases_port'),
+
+    'genblast_rnaseq_support_db_server'  => $self->o('databases_server'),
+    'genblast_rnaseq_support_db_port'    => $self->o('databases_port'),
+
+    'rnaseq_refine_db_server'       => $self->o('databases_server'),
+    'rnaseq_refine_db_port'         => $self->o('databases_port'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release'              => $ENV{ENSEMBL_RELEASE}, # this is the current release version on staging to be able to get the correct database
+    'production_db_server'         => 'mysql-ens-meta-prod-1',
+    'production_db_port'           => '4483',
+
+#    databases_to_delete => ['genblast_db', 'rnaseq_refine_db',],
```

Comment:
This can be uncommented and it should have the data databases that are created during the pipeline so the genblast_rnaseq_*

Reply:
added